x-pack/plugins/security_solution/public/common/components/ml/anomaly/use_anomalies_table_data.ts

```diff
@@ -116,7 +118,7 @@ export const useAnomaliesTableData = ({
       }
     }
 
-    fetchAnomaliesTableData(influencers, criteriaFields, startDate, endDate);
+    fetchAnomaliesTableData(influencers, criteriaFields, startDateMs, endDateMs);
```

Comment:
nit - why not have `startDateMs` and `endDateMs` in the dependencies, maybe you don't need all those `exhaustive-deps` then?

Reply:
You are right. I missed it. I will do it in another PR as we are so close in FF and the test are being flacky.